db/migrate/20200226194012_add_notification_settings.rb

```diff
@@ -0,0 +1,22 @@
+class AddNotificationSettings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notification_settings do |t|
+      t.integer :account_id
+      t.integer :user_id
+      t.integer :email_flags, null: false, default: 0
+
+      t.timestamps
+    end
+
+    add_index :notification_settings, [:account_id, :user_id], unique: true, name: 'by_account_user'
+
+    ::User.find_in_batches do |users_batch|
+      users_batch.each do |user|
+        user_notification_setting = user.notification_settings.new(account_id: user.account_id)
+        user_notification_setting.conversation_assignment = true
+        user_notification_setting.new_conversation_when_auto_assign_disabled = true
```

Comment:
We can switch this setting to `conversation_creation` and make this flag false by default, since this can generate a lot of notification emails. So user will have to explicitly enable this.